app/styles/_material_icons.scss

```diff
@@ -0,0 +1,22 @@
+.material-icons {
+  font-family: 'Material Icons';
```

Comment:
Properties should be ordered display, font-family, font-feature-settings, font-size, -moz-osx-font-smoothing, -webkit-font-smoothing, font-style, font-weight, height, line-height, text-rendering, text-transform, width<br>Prefer double-quoted strings